std/didexchange/invitation/models.go

```diff
@@ -1,67 +1,67 @@
-// Taken from aries-framework-go, and heavily modified. The idea is to replace
-// these with the aries-framework-go when it's ready. Until now we use our own
-// minimalistic solution.
-
-// Package invitation is for invitation data model. It includes the JSON struct.
 package invitation
 
 import (
-	"github.com/findy-network/findy-common-go/std/decorator"
-)
-
-// Invitation model
-//
-// Invitation defines DID exchange invitation message
-// https://github.com/hyperledger/aries-rfcs/tree/master/features/0023-did-exchange#0-invitation-to-exchange
-type Invitation struct {
-	// the Image URL of the connection invitation
-	ImageURL string `json:"imageUrl,omitempty"`
-
-	// the Service endpoint of the connection invitation
-	ServiceEndpoint string `json:"serviceEndpoint,omitempty"`
+	"strings"
 
-	// the RecipientKeys for the connection invitation
-	RecipientKeys []string `json:"recipientKeys,omitempty"`
-
-	// the ID of the connection invitation
-	ID string `json:"@id,omitempty"`
-
-	// the Label of the connection invitation
-	Label string `json:"label,omitempty"`
+	"github.com/hyperledger/aries-framework-go/pkg/vdr/fingerprint"
+	"github.com/lainio/err2/try"
+	"github.com/mr-tron/base58"
+)
 
-	// the DID of the connection invitation
-	DID string `json:"did,omitempty"`
+type DIDExchangeVersion int
 
-	// the RoutingKeys of the connection invitation
-	RoutingKeys []string `json:"routingKeys,omitempty"`
+const (
+	DIDExchangeVersionV0 DIDExchangeVersion = iota
```

Comment:
could this be:
```go
.. = 10 * iota 
```
I don't know the semantics behind it or what happens when V3 comes around, but it would work with that. But maybe there is something else I don't know.

Reply:
Decided to leave out the use of iota as I want to export the values and those should be unchanged.